### PR TITLE
example of ruby script for API integration

### PR DESCRIPTION
**Pull request by Chekushkin**

### CODE REVIEW

File: ruby/Saltedge.rb [hunk 1]
```diff
@@ -0,0 +1,52 @@
+require "json"
+require "base64"
+require "openssl"
+require "digest/sha1"
+require "rest-client"
+require "active_support"
+require "active_support/core_ext"
+
+class Saltedge  attr_reader :expires_at, :client_id, :service_secret, :url, :method, :params
```
Comment (alisnic) [inline ruby/Saltedge.rb]: indent attr reader inside class

File: ruby/Saltedge.rb [hunk 2]
```diff
@@ -0,0 +1,52 @@
+require "json"
+require "base64"
+require "openssl"
+require "digest/sha1"
+require "rest-client"
+require "active_support"
+require "active_support/core_ext"
+
```
Comment (alisnic) [inline ruby/Saltedge.rb]: add comments here on how to use this

File: ruby/README.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+**Basic Info**
```
Comment (alisnic) [inline ruby/README.md]: look at this markdown in preview, you need to add blank lines

File: ruby/Saltedge.rb (hunk 1 above)
Comment (Chekushkin) [inline ruby/Saltedge.rb]: it's a typo.

**Pull request closed without merging**